core/server/master/src/main/java/alluxio/master/block/meta/MasterWorkerInfo.java

```diff
@@ -366,17 +331,18 @@ public boolean isRegistered() {
    * @return the map from tier alias to lost storage paths in this worker
    */
   public Map<String, List<String>> getLostStorage() {
-    return new HashMap<>(mLostStorage);
+    return new HashMap<>(mUsage.mLostStorage);
   }
 
   /**
    * @return true if this worker has lost storage, false otherwise
    */
   public boolean hasLostStorage() {
-    return mLostStorage.size() > 0;
+    return mUsage.mLostStorage.size() > 0;
   }
 
   @Override
+  // TODO(jiacheng): is this locking
```

Comment:
Actually when we are not locking on `toString` properly....
https://github.com/Alluxio/alluxio/blob/9f5673eccf6452c13493ae3bcd38f6203714f16a/core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java#L912
@ggezer